app/services/chat_service.py

```
@@ -128,18 +114,39 @@ async def _openai_token_generator(
             await asyncio.sleep(0)
 
 
-async def _retrieve_context(query: str) -> List[Tuple[str, Dict[str, Any]]]:
+async def _retrieve_context(query: str, user_id: str = None, session: AsyncSession = None) -> List[Tuple[str, Dict[str, Any]]]:
```

Pylint (3.3.7)

[convention] Line too long (127/100)

(C0301)